python/GafferSceneTest/InteractiveRenderTest.py

```diff
@@ -1089,7 +1089,7 @@ def testAddLight( self ) :
 		self.uiThreadCallHandler.waitFor( 1 )
 
 		c = self._color3fAtUV( s["catalogue"], imath.V2f( 0.5 ) )
-		self.assertEqual( c / c[0], imath.Color3f( 1, 1, 0 ) )
+		self.assertTrue( ( c / c[0] ).equalWithAbsError( imath.Color3f( 1, 1, 0 ), 0.08 ) )
```

Comment:
Possibly, and perhaps the [many lights sampling](https://developer.blender.org/T77889) patch will change things again in a later version.